CHANGELOG.md

```diff
@@ -10,6 +10,51 @@ Format `<github issue/pr number>: <short description>`.
 * [#2922](https://github.com/kroxylicious/kroxylicious/pull/2922): build(deps): bump kafka.version from 4.1.0 to 4.1.1
 * [#1318](https://github.com/kroxylicious/kroxylicious/issues/1318): Add FilterContext#topicNames to enable filters to retrieve names for topic ids
 * [#2821](https://github.com/kroxylicious/kroxylicious/pull/2821): Fix OauthBearerValidationFilter unnecessarily copying the authentication bytes from an incoming request to a failed response 
+* [#2893](https://github.com/kroxylicious/kroxylicious/pull/2893): Add Subject, replace FilterContext#clientSaslAuthenticationSuccess
+* [#2899](https://github.com/kroxylicious/kroxylicious/pull/2898): Add SaslSubjectBuilder API
+* [#2899](https://github.com/kroxylicious/kroxylicious/pull/2899): Add the Authorizer API
+* [#2903](https://github.com/kroxylicious/kroxylicious/pull/2903): Add an ACL Authorizer implementation
+* [#2909](https://github.com/kroxylicious/kroxylicious/pull/2909): Add an Authorizer Filter that can authorize Topic operations
+* [#2913](https://github.com/kroxylicious/kroxylicious/pull/2913): Add pluggable mTLS Subject publishing
```

Comment:
Can we make this read a little more symmetrically with the SASL one? Since they're basically two different ways of building Subjects.